### PR TITLE
Add merges

### PR DESCRIPTION
**Pull request by sagemlee**
## Pull request checklist
Please check if your PR fulfills the following requirements:
- [ ] Lint (`npm run lint`) has passed locally and any fixes were made for failures

## Pull request type
 
 
Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Testing
- [x] Other (please describe):  Added migrations to our Users model

## What is the current behavior?

- User currently only has email and role as an attribute

## What is the new behavior?

- User now will have description, shipDate, shippingAddress, and imageUrl

## Does this introduce a breaking change?
- [ ] Yes
- [ ] No

## Other information

Team will have to migrate the database to their local machines. Delivery info migrations WIP.

### CODE REVIEW

File: code/api/src/modules/user/model.js [hunk 1]
```diff
@@ -14,6 +14,18 @@ module.exports = function(sequelize, DataTypes) {
     },
     role: {
       type: DataTypes.TEXT
+    },
```
Comment (sagemlee) [inline code/api/src/modules/user/model.js:17]: Added these new columns into our model,  we will still have to adjust our mutations and our queries in future user stories.

File: code/package.json [hunk 2]
```diff
@@ -23,8 +23,15 @@
     "url": "https://github.com/atulmy/crate/issues"
   },
   "homepage": "https://github.com/atulmy/crate#readme",
-  "dependencies": {},
+  "dependencies": {
```
Comment (sagemlee) [inline code/package.json:26]: This was added in when I installed knex, we should no longer need this.

File: .gitignore [hunk 3]
```diff
@@ -1 +1,2 @@
-.idea/
\ No newline at end of file
+.idea/
+/code/api/src/config/database.json
```
Comment (sagemlee) [inline .gitignore:2]: This was added in so github doesnt know my postgres password

File: code/api/src/migrations/20200825025333-addDescriptionColumn.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => { 
+    return queryInterface.addColumn(
```
Comment (sagemlee) [inline code/api/src/migrations/20200825025333-addDescriptionColumn.js:5]: Migrations use the Sequelize library. Here is a link to documentation if you should need it for your own migrations: https://sequelize.readthedocs.io/en/latest/docs/migrations/

File: code/api/src/config/database.json [hunk 5]
```diff
@@ -1,18 +0,0 @@
-{
```
Comment (KyleWong2510) [inline code/api/src/config/database.json:1]: Was this file not needed?

File: code/api/src/modules/user/model.js [hunk 6]
```diff
@@ -14,6 +14,18 @@ module.exports = function(sequelize, DataTypes) {
...
+    shippingAddress: {
+      type: DataTypes.DATEONLY
+    },
+    shipDate: {
+      type: DataTypes.DATEONLY
+    },
+    imageUrl: {
+      type: DataTypes.STRING
     }
   })
```
Comment (KyleWong2510) [inline code/api/src/modules/user/model.js:31]: Nice job.  Glad to know what this thing looks like now!  Now we should be able to make POST requests?

File: code/package-lock.json [hunk 7]
```diff
@@ -0,0 +1,2493 @@
...
+      "resolved": "https://registry.npmjs.org/yargs-parser/-/yargs-parser-11.1.1.tgz",
+      "integrity": "sha512-C6kB/WJDiaxONLJQnF8ccx9SEeoTTLek8RVbaOIsrAUS8VrBEXfmeSnCZxygc+XC2sNMBIwOOnfcxiynjHsVSQ==",
+      "dev": true,
+      "requires": {
+        "camelcase": "^5.0.0",
+        "decamelize": "^1.2.0"
+      }
+    }
+  }
+}
```
Comment (KyleWong2510) [inline code/package-lock.json:2493]: Where did all these additions come from?  Just curious more than anything else.

File: code/package.json (hunk 2 above)
Comment (KyleWong2510) [reply code/package.json:26]: Is this comment regarding line 29?

Comment (sagemlee) [reply code/package.json:26]: Regarding the whole block.

File: code/api/src/config/database.json (hunk 5 above)
Comment (sagemlee) [reply code/api/src/config/database.json:1]: This is the file that includes my postgres username and password, we should all have unique ones, so I decided not include mine.

File: code/package-lock.json (hunk 7 above)
Comment (sagemlee) [reply code/package-lock.json:2493]: I added them manually in. In rails, it does this for us, so I was surprised I had to do it. I just updated it so now it reflects what our User table looks like.

File: code/api/src/modules/user/model.js (hunk 6 above)
Comment (sagemlee) [reply code/api/src/modules/user/model.js:31]: You should be able to for email once Alex's user story is done. The other attributes wont work till we get through those user stories on our end.

**Comment by sagemlee** [comment]
Everything should still work as it did, I can add default values in, but right now everything defaults to null.

**Pull request merged**